### PR TITLE
feat: implement interfaces of compressor

### PR DESCRIPTION
**Pull request by myk1343**
add dtkcompressor

Log: implement interfaces of compressor

### CODE REVIEW

**Review by uniontechzhangsong — COMMENTED** [review]
+1

**Review by dandan-y — COMMENTED** [review]
+1

File: debian/libdtkcompressor.install [hunk 1]
```diff
@@ -0,0 +1 @@
+usr/lib/*/libdtkcompressor.so.*
```
Comment (Groveer) [inline debian/libdtkcompressor.install:1]: 确认一下是否需要拆包，若需要 其他子项目的install文件也要同步加上

**Comment by Groveer** [comment]
+1

**Review by xzl01 — APPROVED** [review]

**Comment by toberyan** [comment]
+1

**Pull request merged**